src/eel_driver.py

```diff
@@ -5,12 +5,41 @@
 import logging
 from logging.handlers import RotatingFileHandler
 from pathlib import Path
+import json
+import requests
+import os
+import psutil
+import base64
 
 import eel
 
 from champselect import websockets
 from utils import ddragon, runes, sum_spells, summoner_info, champ_identifier
 
+@eel.expose
+def get_ranked_names():
+    # Find the LeagueClient.exe process and extract the required information
+    for process in psutil.process_iter():
```

Comment:
So as you can see from the way we write our code, we use specific files for our functions and then call them in order to keep the eel_driver.py file clean. A good place for this function is in a file in the utilities folder that you can import and call